src/components/EditableHeading/EditableHeading.jsx

```diff
@@ -239,6 +242,7 @@ EditableHeading.propTypes = {
   size: PropTypes.oneOf([EditableHeading.sizes.SMALL, EditableHeading.sizes.MEDIUM, EditableHeading.sizes.LARGE]),
   displayPlaceholderInTextMode: PropTypes.bool,
   suggestEditOnHover: PropTypes.bool,
+  brandFont: PropTypes.bool,
```

Comment:
Could you please add a comment here for the storybook explaining what does this prop? It's not obvious what does this prop do

Reply:
Thanks for this remark, corrected.